bootstrap/templates/kubernetes/flux/repositories/helm/openebs.yaml.j2

```diff
@@ -2,8 +2,8 @@
 apiVersion: source.toolkit.fluxcd.io/v1beta2
 kind: HelmRepository
 metadata:
-  name: democratic-csi
+  name: openebs
```

Comment:
You will also need to update the `kustomization.yaml` file in the same directory.